components/ws-daemon/pkg/cgroup/plugin_process_priority_v2.go

```diff
@@ -42,8 +45,16 @@ const (
 	NumberOfProcessesToStopApplying = 5
 )
 
+type OOMScoreAdjConfig struct {
+	Enabled bool `json:"enabled"`
+	Tier1   int  `json:"tier1"`
+	Tier2   int  `json:"tier2"`
+}
+
 type ProcessPriorityV2 struct {
 	ProcessPriorities map[ProcessType]int
+	OOMScoreAdj       map[ProcessType]int
+	EnableOOMScoreAdj bool
```

Comment:
What's the difference between EnableOOMScoreAdj and `OOMScoreAdjConfig.Enabled`?

Reply:
EnableOOMScoreAdj enables setting the oom scores and OOMScoreAdj describes which values to set for each process type.

Reply:
Can we remove the `OOMScoreAdjConfig.Enabled`?

Because we could use `EnableOOMScoreAdj` to decide whether to set the OOM scores or not.

Reply:
`EnableOOMScoreAdj` gets its value from `OOMScoreAdjConfig.Enabled`

Reply:
@jenting Does that answer your question?

Reply:
Hmmm, so the `EnableOOMScoreAdj = OOMScoreAdjConfig.Enabled`?

Reply:
Yes, OOMScoreAdjConfig is the struct that the configuration is read into and EnableOOMScoreAdj is where it ends up.